.github/workflows/e2e-tests-workflow.yml

```diff
@@ -11,6 +11,8 @@ env:
   AD_OPENSEARCH_DASHBOARDS_PLUGIN_NAME: anomalyDetectionDashboards
   OPENSEARCH_DOCKER_IMAGE: opensearchstaging/opensearch
   DASHBOARDS_DOCKER_IMAGE: opensearchstaging/opensearch-dashboards
+  OPENSEARCH_DOCKER_TAG: 1.0.0-rc1
+  DASHBOARDS_DOCKER_TAG: 1.0.0-rc1
```

Comment:
Can we assume those two are always the same?  If so, we can have just one tag.

Reply:
sure - I'll combine into a single field.